script/cms/test-bundle-transformers.js

```diff
@@ -66,9 +65,16 @@ const {
   print: printIndex,
   bundle,
   help,
-  transformUnpublished,
+  'transform-unpublished': transformUnpublished,
```

Comment:
After testing it doesn't seem like the default value for `transform-unpublished` is being set to true.